TESTS/mbed_drivers/timeout/timeout_tests.h

```diff
@@ -305,14 +305,14 @@ void test_deepsleep(void)
     LowPowerTimer timer;
 
     /*
-     * Since deepsleep() may shut down the UART peripheral, we wait for 10ms
+     * Since deepsleep() may shut down the UART peripheral, we wait for 20ms
      * to allow for hardware serial buffers to completely flush.
 
      * This should be replaced with a better function that checks if the
      * hardware buffers are empty. However, such an API does not exist now,
      * so we'll use the wait_ms() function for now.
      */
-    wait_ms(10);
+    wait_ms(20);
```

Comment:
As @0xc0170 pointed out, a comment with a short explanation would be very useful here.

Reply:
@0xc0170 @fkjagodzinski I add comment for enlarging wait time in https://github.com/ARMmbed/mbed-os/pull/7224/commits/39a6fa960952d16cfedccdc3633a260647c9f0d3 (override original one).